Source/Scene/Expression.js

```diff
@@ -43,6 +43,19 @@ define([
         }
     };
 
+    var unaryFunctions = {
+        abs : Math.abs,
+        sqrt : Math.sqrt,
+        cos : Math.cos,
+        sin : Math.sin,
+        tan : Math.tan,
+        acos : Math.acos,
+        asin : Math.asin,
+        atan : Math.atan,
+        radians : CesiumMath.toRadians(),
+        degrees : CesiumMath.toDegrees()
```

Comment:
What you're doing in the screenshot is fine (though we alphabetize the order in the first first array instead of the second, so ../Core/Math should be after isArray). The code pushed to github has a bunch of unnecessary includes in it.